docs/tendermint-core/running-in-production.md

```diff
@@ -120,15 +120,15 @@ whenever the node is syncing or not, what height it is on, etc.
 curl http(s)://{ip}:{rpcPort}/status
 ```
 
-`/dump_consensus_state` will give you a detailed overview of the consensus
+`/dump-consensus-state` will give you a detailed overview of the consensus
```

Comment:
did we modify RPC endpoints too?

Reply:
nope, good catch. I tried to sift through which were RPC and which were config but might have missed some